mining/worker.go

```diff
@@ -214,7 +219,12 @@ func (w *DefaultWorker) Mine(ctx context.Context, base types.TipSet, nullBlkCoun
 			return false
 		}
 		copy(proof[:], prChRead[:])
-		ticket = consensus.CreateTicket(proof, w.minerAddr)
+		signer := w.workerSigner
```

Comment:
why not just call `w.workerSignerCreateTicket`?

Reply:
herpty-derp